src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -227,4 +238,16 @@ private static String getIndividualAuthorshipPath(String repoReportDirectory) {
     private static String getIndividualCommitsPath(String repoReportDirectory) {
         return repoReportDirectory + "/commits.json";
     }
+
+    public static void setEarliestSinceDate(Date newEarliestSinceDate) {
+        if (earliestSinceDate == null || earliestSinceDate.after(newEarliestSinceDate)) {
```

Comment:
I think normally when people look for the earliest date/object, they usually compare who comes before, instead of after, in multiple comparison context. To standardize with that convention, let's change it to use `before` instead

Reply:
@eugenepeh done